tox.ini

```diff
@@ -40,11 +40,11 @@ basepython = python3.6
 commands_pre =
     mkdir -p {toxinidir}/parts/lint
 commands =
-    isort --check-only --diff --recursive {toxinidir}/src setup.py
+    isort --check-only --diff {toxinidir}/src setup.py
     - flake8 --format=html src setup.py
     flake8 src setup.py
 deps =
-    isort
+    isort > 5.0
```

Comment:
You don't need to pin here. It will grab the latest.

Reply:
Unfortunately, only if the tox env is re-created. This ensures the re-creation. This can be removed later, of course.